pset-iterators.js

```diff
@@ -66,6 +72,14 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const presidentsNamedJames = presidents.filter(president => {
+    if(president.president.includes("James") === true){
+      return president
+    }
+})
+
+console.log(presidentsNamedJames)
```

Comment:
@nmikhol Good job here using the filter method to find all the presidents whose names includes James. Good work using the conditional to return the president if the James is included in the name

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L62)